[Bug] Vite returns No loader is configured for ".po" files

### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and none of them match my problem.

### Product Variant

Cloud (http://rxresu.me)

### Current Behavior

When checking out the project, following the setup in adherence to the CONTRIBUTING.md and then running the development server for the first time, the following issue occurs for me: 
```
Error:   Failed to scan for dependencies from entries:
  /Users/sergejkasper/proj/Reactive-Resume/apps/client/index.html

  ✘ [ERROR] No loader is configured for ".po" files: apps/client/src/locales/af-ZA/messages.po

    apps/client/src/libs/lingui.ts:10:38:
      10 │ ...t { messages } = await import(`../locales/${locale}/messages.po`);
         ╵          
```
This is an abbreviated stack trace. The same error is listed for all languages in the console.

### Expected Behavior

The project should start correctly, when following the instructions in the CONTRIBUTING.md

### Steps To Reproduce

1. Fork and Clone the Repository
2. Install dependencies
3. Copy .env.example to .env
4. Fire up all the required services through Docker Compose
5. Run the development server


### What browsers are you seeing the problem on?

_No response_

### What template are you using?

None

### Anything else?

I was able to fix the issue by polyfilling amending the vite config. PR coming.

--- a/apps/client/vite.config.ts
+++ b/apps/client/vite.config.ts
@@ -22,6 +22,14 @@ export default defineConfig({
     fs: { allow: [searchForWorkspaceRoot(process.cwd())] },
   },
 
+  optimizeDeps: {
+    esbuildOptions: {
+      loader: {
+        ".po": "text",
+      },
+    },
+  },
+
   plugins: [
     react({
       plugins: [["@lingui/swc-plugin", {}]],

